### PR TITLE
Added passing down additional attributes to component

### PR DESCRIPTION
**Pull request by BabkinAleksandr**
Issue #20 

Btw, it could pass any additional props, not just html attributes.

### CODE REVIEW

File: src/react-swipe.js [hunk 1]
```diff
@@ -176,17 +176,33 @@ class ReactSwipe extends Component {
...
       <this.props.tagName
         ref={node => this.swiper = node}
         onMouseDown={ this._onMouseDown }
         onTouchStart={ this._handleSwipeStart }
         onTouchEnd={ this._handleSwipeEnd }
-        className={ this.props.className }
-        style={ this.props.style }
+        className={ className }
+        style={ style }
+        { ...props }
```
Comment (leandrowd) [inline src/react-swipe.js:202]: You would probably want to apply move {...props} to before all other props are declared so required props are not overridden.

Comment (alekskin) [reply src/react-swipe.js:202]: I have declared all required props (except tolerance) above the `return`, so `props` variable contains only that properties, that is not used in this component.
So, there should be no collisions.

Comment (alekskin) [reply src/react-swipe.js:202]: @leandrowd Is all ok? If is, please, merge PR

Comment (leandrowd) [reply src/react-swipe.js:202]: Merged and published to npm...

**Pull request merged**